### DISCUSSION TITLE
Email using STMP without SendGrid

### DISCUSSION THREAD

**Discussion by hbc-acai** [apache/airflow#34374] [Q&A] [answered]
Hello community, 

Is it possible to send emails using our on-premise STMP server without SendGrid? 

In the [email config document](https://airflow.apache.org/docs/apache-airflow/stable/howto/email-config.html), the Using Default SMTP section is under the SendGrid section. This makes me think I have to use SendGrid. 

Thanks,
Adam

**Comment by potiuk** [comment, marked as answer]
yes you can. the example is for sendgrid's "default SMTP" interface

**Reply by hbc-acai** [reply]
OK great. Thank you.
